packages/xo-web/src/common/tags.js

```diff
@@ -107,14 +157,38 @@ export default class Tags extends Component {
           <span onClick={this._startEdit} style={ADD_TAG_STYLE}>
             <Icon icon='add-tag' />
           </span>
+        ) : hasSelectExistingTag ? (
+          <span
+            className='form-inline'
+            onBlur={this._stopEditIfUnfocused}
+            onFocus={this._focus}
+          >
+            <span className='input-group'>
+              <input
+                autoFocus
+                className='form-control'
+                onKeyDown={this._onKeyDown}
+                style={INPUT_STYLE}
+                type='text'
+              />
+              <span className='input-group-btn'>
```

Comment:
Doesn't it need to be a `div`?

Reply:
To keep it as close as possible from the doc. If it's not possible for some reason and you confirm that it doesn't break anything, we can keep the span.